### ISSUE TITLE
Invalid version string on composer installation

### ISSUE THREAD

**Issue by davidsonalencar** [francoispluchino/composer-asset-plugin#92]
I tried installing with the animate.css package and not work.

Error message:

```
  [UnexpectedValueException]
  Invalid version string "~3.2.0"
```

My composer.json:

```
    "require": {
    ...
        "bower-asset/animate.css": "3.2.0",
    ...
    },
```

bower.json file of the https://github.com/daneden/animate.css

```
{
    "name": "animate.css",
    "version": "~3.2.0",
    "main": "./animate.css"
}
```

It's not work because of the "~"?

**Comment by francoispluchino** [comment]
Fixed by a54431994e8524dc09730e7985a9d7c9a9d636c0.

**Comment by davidsonalencar** [comment]
Thanks!
